packages/page-accounts/src/modals/Create.tsx

```diff
@@ -225,24 +249,30 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
   ));
 
   const _onChangePath = useCallback(
-    (newDerivePath: string) => setAddress(
-      updateAddress(seed, newDerivePath, seedType, pairType)
-    ),
+    (newDerivePath: string) =>
+      setAddress(
+        updateAddress(seed, newDerivePath, seedType, pairType)
+      )
+    ,
     [pairType, seed, seedType]
   );
 
   const _onChangeSeed = useCallback(
-    (newSeed: string) => setAddress(
-      updateAddress(newSeed, derivePath, seedType, pairType)
-    ),
+    (newSeed: string) =>
+      setAddress(
+        updateAddress(newSeed, derivePath, seedType, pairType)
+      )
+    ,
     [derivePath, pairType, seedType]
   );
 
   const _onChangePairType = useCallback(
-    (newPairType: PairType) => setAddress(
-      updateAddress(seed, '', seedType, newPairType)
-    ),
-    [seed, seedType]
+    (newPairType: PairType) =>
+      setAddress(
+        updateAddress(seed, derivePath, seedType, newPairType)
```

Comment:
This is going to cause some issues. We explicitly cleared the path on all derivations previously. So this aligns with what is done in the initial state, where the path is set to the original version.

Reply:
The problem is that this function is called when clicking on the "advanced" dropdown and we need the derivation to be set to a valid derivationPath or the derivation won't work

Reply:
That is fine - but my point is that it is an ethereum-specific issue - so it should have ethereum-specific logic.

 I'm not against changing it globally, but then we raise the testing surface for this completely, i.e. it means all the types and incarnations needs to be completely re-tested since this changes the existing in-place behavior.
 
 So my comments here are all around not changing existing code flow or behavior, but rather to add the the option for ethereum.

Reply:
sounds good